src/App.js

```diff
@@ -1,11 +1,19 @@
 import React from "react";
```

Comment:
remove all the warnings from console. For firebase to auto deploy we need a clean console without warnings and errors.